src/features/GermplasmSort.feature

```diff
@@ -0,0 +1,82 @@
+		When user selects 'Save' button in Programs page
+		When user pause for "10" seconds
+		When user navigates to Program Selection page
+		When user selects "<Program Name>" on program-selection page
+		When user selects "Program Management" in navigation
+		When user selects "Users" tab
+		When user clicks 'New User' button
+		When user sets "Cucumber Breeder" in Name field of User
+		When user sets "cucumberbreeder@mailinator.com" in Email field of User
+		When user sets "breeder" in Role dropdown of User
+		When user click 'Save' button in User
+		When user pause for "10" seconds
+		When user close notification pop-up
+		When user logs out
+		#Login as Cucumber Breeder and go to Snacks
+		Given user logs in as "Cucumber Breeder"
+		When user selects "<Program Name>" on program-selection page
+		And user selects "Import Data" in navigation
+		And user uploads Germplasm "Germ-syno.xls" file
+		And user selects 'Import' button
+		When user sets "GermplasmSort" in List Name field of import page
+		When user sets "GermplasmSort" in List Description field of import page
+		And user selects "Confirm" button
+		And user pause for "10" seconds
+		And user selects "Germplasm" in navigation
+		Then user can see All Germplasm records exist on Germplasm page
+		When user clicks on GID sort descending
+		Then user can see "11" GID in first line
+		When user clicks on GID sort ascending
+		Then user can see "1" GID in first line
+		When user clicks on "Name" of Germplasm table
+		When user clicks on Name sort descending
+		Then user can see "unknown" Name in first line
+		When user clicks on Name sort ascending
+		Then user can see "Germplas123" Name in first line
+		When user clicks on "Breeding Method" of Germplasm table
+		When user clicks on Breeding Method sort descending
+		Then user can see "Three-way cross" Breeding Method in first line
+		When user clicks on Breeding Method sort ascending
+		Then user can see "Backcross" Breeding Method in first line
+		When user clicks on "Source" of Germplasm table
+		When user clicks on Source sort descending
+		Then user can see "Cross" Source in first line
+		When user clicks on Source sort ascending
+		Then user can see "Cross" Source in first line
+		When user clicks on "Female Parent GID" of Germplasm table
+		When user clicks on Female Parent GID sort descending
+		Then user can see "6" Female Parent GID in first line
+		When user clicks on Female Parent GID sort ascending
+		Then user can see "" Female Parent GID in first line
+		When user clicks on "Male Parent GID" of Germplasm table
+		When user clicks on Male Parent GID sort descending
+		Then user can see "6" Male Parent GID in first line
+		When user clicks on Male Parent GID sort ascending
+		Then user can see "" Male Parent GID in first line
+		When user clicks on "Created Date" of Germplasm table
+		When user clicks on Created Date sort descending
+		Then user can see "2022-10-27" Created Date in first line
```

Comment:
Would it be better to have data that would validate the sorting rather than being 2022-10-27 for both?

Reply:
I think we can't test the date sort. Every time the germplasm is uploaded, it will use the current date.
Besides, the programs in github action are always new.
@ShahanaMu - do you think we can remove the Create Date sort validation?

Reply:
@ShahanaMu has confirmed that we can remove the validation for Create Date.